leader-election-impl/build.gradle

```diff
@@ -11,3 +11,13 @@ dependencies {
   testCompile group: 'org.mockito', name: 'mockito-core'
   testCompile group: 'org.assertj', name: 'assertj-core'
 }
+
+configurations {
+    testArtifacts {
+       extendsFrom testRuntime
+    }
+}
+
+artifacts {
```

Comment:
Copypaste from reviewable: 

Hmm, I get Could not set unknown property 'classifier' for DefaultProjectDependency{dependencyProject='project ':leader-election-impl'', configuration='default'} of type org.gradle.api.internal.artifacts.dependencies.DefaultProjectDependency.. If I'm not wrong [this looks like it applies to external dependencies](https://docs.gradle.org/current/userguide/dependency_management.html)?